### ISSUE TITLE
PaddleOCR微调，微调数据以文本密集、长句为主，发现 Loss 比较高无法下降

### ISSUE THREAD

**Issue by yilong2001** [PaddlePaddle/PaddleOCR#11969]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
  PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
  NAME="Debian GNU/Linux"
  VERSION_ID="11"
  VERSION="11 (bullseye)"
  VERSION_CODENAME=bullseye
  ID=debian
  Python 3.10.14
  GPU Compute Capability: 8.6, Driver API Version: 12.2, Runtime API Version: 12.0

- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
  paddle-bfloat         0.1.7
  paddle2onnx           1.2.0
  paddlefsl             1.1.0
  paddlenlp             2.7.2
  paddlepaddle-gpu      2.6.1.post120
  PaddleOCR          2.7.0

- 运行指令/Command Code：
python tools/train.py -c configs/det/ch_PP-OCRv3/ch_PP-OCRv3_det_dml-g.yml -o Architecture.Models.Student.pretrained=ch_PP-OCRv3_det_distill_train/teacher.pdparams Architecture.Models.Student2.pretrained=ch_PP-OCRv3_det_distill_train/teacher.pdparams Global.save_model_dir=./output/ Global.pretrained_model=ResNet50_vd_ssld_pretrained.pdparams

- 完整报错/Complete Error Message：
  几种微调方法都做了尝试、LR调整做了尝试、batch_size用4、8两种做了尝试（24G的GPU内容），发现微调效果都不太好
（1）基于CML蒸馏方法的finetune训练：训练1000轮，loss 最小0.8左右，效果比标准的PP_OCRv3 det模型差
（2）基于PP-OCRv3轻量检测模型的finetune训练：训练1000轮，loss 最小0.8左右，效果比标准的PP_OCRv3 det模型差
（3）基于DML蒸馏方法的finetune训练：训练 1200轮，loss 最小1.2左右，如下是相关日志。结果比较奇怪：
微调效果实测：短文本无法检测，长文本能检测的连续性稍微好，但是文本宽度更窄，识别效果更差。
[2024/04/18 01:01:54] ppocr INFO: epoch: [1200/1200], global_step: 137998, lr: 0.000000, dml_thrink_maps_0: 0.005081, loss: 1.276292, DBLoss_Student_loss_shrink_maps: 0.360316, DBLoss_Student_loss_threshold_maps: 0.195564, DBLoss_Student_loss_binary_maps: 0.071449, DBLoss_Student_loss_cbn: 0.000000, DBLoss_Student2_loss_shrink_maps: 0.359688, DBLoss_Student2_loss_threshold_maps: 0.193915, DBLoss_Student2_loss_binary_maps: 0.071279, DBLoss_Student2_loss_cbn: 0.000000, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.92222 s, avg_samples: 4.0, ips: 4.33735 samples/s, eta: 0:00:01
[2024/04/18 01:01:56] ppocr INFO: epoch: [1200/1200], global_step: 138000, lr: 0.000000, dml_thrink_maps_0: 0.005075, loss: 1.276517, DBLoss_Student_loss_shrink_maps: 0.365204, DBLoss_Student_loss_threshold_maps: 0.195564, DBLoss_Student_loss_binary_maps: 0.072151, DBLoss_Student_loss_cbn: 0.000000, DBLoss_Student2_loss_shrink_maps: 0.364065, DBLoss_Student2_loss_threshold_maps: 0.195290, DBLoss_Student2_loss_binary_maps: 0.072020, DBLoss_Student2_loss_cbn: 0.000000, avg_reader_cost: 0.00009 s, avg_batch_cost: 0.80950 s, avg_samples: 3.5, ips: 4.32366 samples/s, eta: 0:00:00

对于这种微调的结果，是否可能是标注的数据存在问题？

**Comment by UserWangZz** [comment]
可以尝试推理一下看看是否有hard case，如果有建议增加一些这种场景的数据

**Comment by yilong2001** [comment]
{
        "img": "202312070001_crop_13.jpg",
        "gen": [
            "送检医师：柏维昌",
            0.9480968117713928
        ],
        "label": "送检医师：柏继昌"
    }
    您好，对于微调后的模型，预测错误的 case，增加那些数据效果更好？
    比如：202312070001_crop_13.jpg 做一些数据增广，还是识别错误：“柏继昌” ，必须使用更多的真实数据？

**Comment by yilong2001** [comment]
根据 @UserWangZz 的问题做了 close处理，但issue并未解决。
{
    "img": "202312070001_crop_13.jpg",
    "gen": [
        "送检医师：柏维昌",
        0.9480968117713928
    ],
    "label": "送检医师：柏继昌"
}
您好，对于微调后的模型，预测错误的 case，增加那些数据效果更好？
比如：202312070001_crop_13.jpg 做一些数据增广，还是识别错误：“柏继昌” ，必须使用更多的真实数据？
